content/sips/sip-420.md

```diff
@@ -0,0 +1,105 @@
+---
+sip: 420
+title: Protocol Owned SNAX
+network: All
+status: Draft
+type: Governance
+author: Kain Warwick (@kaiynne)
+implementor: Synthetix Core Contributors
+release: (Release Name)
+implementation-date: TBC
+proposal: <snapshot.org proposal link> (*optional)
```

Comment:
please drop this, only added when we have a link

Reply:
ok